src/field/arena.ts

```diff
@@ -921,80 +921,11 @@ export class Arena {
     globalScene.loadBgm(this.bgm);
   }
 
+  /**
+   * @returns the bgmLoopPoint of a biome's associated bgm
```

Comment:
```suggestion
   * @returns the {@linkcode Biome.bgmLoopPoint | loop point} of a biome's associated bgm
```
Might need a tsdoc import, haven't checked.